src/ocm/components/clusterConfiguration/review/ReviewSummary.tsx

```diff
@@ -0,0 +1,330 @@
+import { Table, TableVariant, TableBody, IRow } from '@patternfly/react-table';
+import {
+  Cluster,
+  CpuArchitecture,
+  hasEnabledOperators,
+  isDualStack,
+  OPERATOR_NAME_CNV,
+  OPERATOR_NAME_ODF,
+  ReviewHostsInventory,
+} from '../../../../common';
+import useInfraEnv from '../../../hooks/useInfraEnv';
+import {
+  getDiskEncryptionEnabledOnStatus,
+  getManagementType,
+  getNetworkType,
+  getStackType,
+} from '../../clusterDetail/ClusterProperties';
+import OpenShiftVersionDetail from '../../clusterDetail/OpenShiftVersionDetail';
+
+const ClusterSummaryExpandable = ({
+  title,
+  children,
+}: {
+  title: string;
+  children?: React.ReactNode;
+}) => {
+  const [isExpanded, setExpanded] = React.useState(true);
+  return (
+    <ExpandableSection
+      toggleContent={<b>{title}</b>}
+      isExpanded={isExpanded}
+      onToggle={() => setExpanded(!isExpanded)}
+      isIndented
+    >
+      {children}
+      <br />
+    </ExpandableSection>
+  );
+};
+
+const ReviewClusterDetailTable = ({ cluster }: { cluster: Cluster }) => {
+  const { infraEnv } = useInfraEnv(cluster.id, cluster.cpuArchitecture as CpuArchitecture);
+
+  const rows = React.useMemo(() => {
+    return [
+      {
+        cells: [
+          { title: 'Cluster address' },
+          {
+            title: (
+              <>
+                {cluster.name || ''}
+                {'.'}
+                {cluster.baseDnsDomain || ''}
+              </>
+            ),
+            props: { 'data-testId': 'cluster-address' },
+          },
+        ],
+      },
+      {
+        cells: [
+          { title: 'OpenShift version' },
+          {
+            title: <OpenShiftVersionDetail cluster={cluster} />,
+            props: { 'data-testId': 'openshift-version' },
+          },
+        ],
+      },
+      {
+        cells: [
+          { title: 'CPU architecture' },
+          { title: <>{cluster.cpuArchitecture}</>, props: { 'data-testId': 'cpu-architecture' } },
+        ],
+      },
+      {
+        cells: [
+          { title: "Hosts' network configuration" },
+          {
+            title: <>{infraEnv?.staticNetworkConfig ? 'Static IP' : 'DHCP'}</>,
+            props: { 'data-testId': 'network-configuration' },
+          },
+        ],
+      },
+      cluster.diskEncryption?.enableOn !== 'none' && {
+        cells: [
+          { title: 'Disk encryption' },
+          {
+            title: getDiskEncryptionEnabledOnStatus(cluster.diskEncryption?.enableOn),
+            props: { 'data-testId': 'disk-encryption' },
+          },
+        ],
+      },
+    ];
+  }, [cluster, infraEnv?.staticNetworkConfig]);
+
+  return (
+    <Table
+      rows={rows as IRow[]}
+      cells={['', '']}
+      variant={TableVariant.compact}
+      borders={false}
+      className="review-table"
+    >
+      <TableBody />
+    </Table>
+  );
+};
+
+const ReviewOperatorsTable = ({ cluster }: { cluster: Cluster }) => {
```

Comment:
It was an easy fix, LVM added. It will only show for SNO clusters instead of ODF.

![image](https://user-images.githubusercontent.com/87187179/208883347-a010f3f5-7ee3-4fbb-a0ae-d5a486fdfe6a.png)